runtime/evaluate.cc

```diff
@@ -323,6 +335,9 @@ class evaluator : public expr_visitor, public stmt_visitor {
       result = std::abs(eval(op->args[0]));
       return;
 
+    case intrinsic::and_then:
+    case intrinsic::or_else: result = eval_short_circuit_op(op); return;
```

Comment:
style nit: I would have expected this to be like

```
    case intrinsic::and_then:
    case intrinsic::or_else: 
        result = eval_short_circuit_op(op); 
        return;
```

Reply:
This is what clang-format does, don't want to fight it.